public/index.html

```diff
@@ -30,6 +30,7 @@
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
+    <div id="root2"></div>
```

Comment:
Ignore this, I was just messing around with adding another div for playing around with DOM manipulation